search_engine/search_engine/webcrawler.py

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/python
+# __init__.py - Make a basic crawler.
+
+import requests
+from html.parser import HTMLParser
+from datetime import datetime
+
+
+class MyHTMLParser(HTMLParser):
+
+    def handle_starttag(self, tag, attrs):
+        if tag == 'a':
+            attr = dict(attrs)
+            if 'href' in attr and not attr['href'].startswith('mailto:') \
+                and not attr['href'].startswith('tel:'):
```

Comment:
@ErisedKT there is a minor style violation here, please rectify it using a tool like autopep8.

Reply:
Fixed it.